lib/db.js

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var P = require('bluebird')
+var extend = require('util')._extend;
```

Comment:
Interesting. This looks undocumented: https://nodejs.org/api/util.html
Not sure if that makes it "risky" and could change/disappear in 0.12 or iojs, etc.

Do we want to use something potentially more stable like [**util-extend**](https://www.npmjs.com/package/util-extend) or [**lodash.assign**](https://www.npmjs.com/package/lodash.assign).